packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -240,15 +242,16 @@ export class GCTelemetryTracker {
 			// Events generated:
 			// InactiveObject_Loaded, SweepReadyObject_Loaded
 			if (usageType === "Loaded") {
-				const { id, fromId, headers, gcConfigs, ...detailedProps } = unrefEventProps;
+				const { id, fromId, headers, gcConfigs, additionalProps, ...detailedProps } =
```

Comment:
nit: Alternative approach would be to get `additionalProps` at the top of the function and spread it into `unrefEventProps`. Then those will naturally end up in `detailedProps` here and below without having to make any more changes.

Reply:
I tried it but it did not work. Other places where event is logged from unrefEventProps failed.